control-plane/roles/monitoring/tasks/main.yaml

```diff
@@ -90,6 +90,13 @@
     - shoot-details.yaml
   when: monitoring_gardener_enabled
 
+- name: Deploy Custom dashboards
+  k8s:
+    definition: "{{ lookup('template', custom_dashboard_folder + item) }}"
+    namespace: "{{ monitoring_namespace }}"
+  loop: "{{ custom_dashboard_file_list }}"
+  when: custom_dashboards_enabled
```

Comment:
Should be added as an empty list to the default variables with an example. Please also add newly introduced variables to the README. Also prefix all variables of this role with `monitoring_`.